### DISCUSSION TITLE
Shred directory contents without deleting the directory itself?

### DISCUSSION THREAD

**Discussion by jgkennedy** [bleachbit/bleachbit#1335] [Q&A] [answered]
Using Windows console interface. I want to shred the contents of a directory without deleting the directory of itself. Can it be done? I tried adding a `\*` to the end but it didn't work.

**Comment by az0** [comment, marked as answer]
This is not a feature, so you need to pass each file or sub-folder as a separate command line argument. If there are not many, you can do it on one command line. A script can make it easier.

It may be easier to just recreate the deleted directory using MKDIR, unless the directory has special properties like compression or permissions for a multi-user system.

**Reply by jgkennedy** [reply]
A MKDIR afterwards seems to be the easiest solution. Thanks.
